components/Icons/EditIcon.tsx

```diff
@@ -2,8 +2,7 @@ import { FC, SVGProps } from 'react'
 
 interface EditIconProps extends SVGProps<SVGSVGElement> {}
 
-const EditIcon: FC<EditIconProps> = props => {
-  return (
+const EditIcon: FC<EditIconProps> = props => (
```

Comment:
Don't forget to erase the curly braces at the end of the component, please check linter before pushing code

CC: @e-azocar please note

Reply:
The linter doesn't notify for that.

Reply:
They should, please ask @Eghinner and @e-azocar help to check why it's not working on your side

Reply:
@JRS-Developer maybe you have not installed lefthook yet, and therefore it does not check the commits. Try npm i, as soon as possible

Reply:
Ok, I executed yarn install and no, it doesn't warning.

![image](https://user-images.githubusercontent.com/77207702/188742403-1b38223b-db36-4161-8d08-cf04a1b0059f.png)

![image](https://user-images.githubusercontent.com/77207702/188742459-a1e8b172-ff1a-4764-aada-bd7dac15792a.png)